engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-traces-modal.html

```diff
@@ -0,0 +1,68 @@
+<div data-extend-template="components/views/bst-modal.html">
+  <h4 data-block="modal-header">Content Host Traces Management</h4>
+
+  <div data-block="modal-body">
+
+    <div class="row">
+      <div class="col-sm-12">
+        <div bst-global-notification></div>
+      </div>
+    </div>
+
+    <div data-extend-template="layouts/partials/table.html">
+      <div data-block="search" ng-show="false"></div>
+      <span data-block="list-actions">
+        <div bst-modal="performViaRemoteExecution()">
+          <div data-block="modal-header" translate>Restart services on the selected Content Hosts ?</div>
```

Comment:
What do you think about the text 'Confirm services restart' for the modal header?

Reply:
Fixed